README.md

```diff
@@ -42,5 +42,5 @@ Example Usage
 ```sh
 # Download latest Kafka distribution(if necessary)
 $ tar zxvf kafka_LATEST_VERSION.tgz
-$ ./kafka_consumer_group_exporter --consumer-group-command-path=./kafka_LATEST_VERSION/bin/kafka-consumer-groups.sh BOOTSTRAP_SERVERS
+$ ./kafka_consumer_group_exporter --consumer-group-command-path=./kafka_LATEST_VERSION/bin/kafka-consumer-groups.sh BOOTSTRAP_SERVER
```

Comment:
> I changed to singular because I thought the help text for kafka-consumer-groups.sh hints that it only takes a single server: 

AFAIK, it takes multiple servers list. The value given by that option will eventually passed to bootstrap.servers configuration of AdminClient, which is capable to accept multiple servers: https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala#L452

I'm not sure why Kafka's help text is in singular form, but I'd think it's better to pass multiple servers for this exporter because of this reason I explained previously:

> preferred to pass multiple bootstrap servers for this exporter argument to make it keep working even when one broker went down.